code/modules/mob/living/carbon/human/human.dm

```diff
@@ -893,3 +893,6 @@
 			Weaken(10)
 		return 1
 	..()
+
+/mob/living/carbon/human/Bump(atom/A)
+	world << "DEBUG: BUMP WITH [A]"

```

Comment:
this may have caused you issues as it wasn't calling ..(); it should call ..() or possibly return ..()